python/python/lance/ray/sink.py

```diff
@@ -174,10 +183,18 @@ def __init__(
         mode: Literal["create", "append", "overwrite"] = "create",
         max_rows_per_file: int = 1024 * 1024,
         use_legacy_format: bool = False,
+        storage_options: Optional[Dict[str, Any]] = None,
         *args,
         **kwargs,
     ):
-        super().__init__(uri, schema=schema, mode=mode, *args, **kwargs)
+        super().__init__(
+            uri,
+            schema=schema,
+            mode=mode,
+            storage_options=storage_options,
+            *args,
+            **kwargs,
+        )
 
```

Comment:
The `_write_fragment()` call below should also receive the `storage_options`, I think.

Reply:
Done 👍